src/objects/Attribute__c.object

```diff
@@ -0,0 +1,252 @@
+    <enableSearch>true</enableSearch>
+    <enableSharing>true</enableSharing>
+    <enableStreamingApi>true</enableStreamingApi>
+    <fields>
+        <fullName>Agency__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The agency providing or certifying the credential or assigning the Attribute.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The agency providing or certifying the credential or assigning the Attribute.</inlineHelpText>
+        <label>Agency</label>
+        <referenceTo>Account</referenceTo>
+        <relationshipLabel>Attributes</relationshipLabel>
+        <relationshipName>Attributes</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Attribute_Type__c</fullName>
+        <description>The type of credential or Attribute.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The type of credential or Attribute.</inlineHelpText>
+        <label>Attribute Type</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Certification</fullName>
+                    <default>false</default>
+                    <label>Certification</label>
+                </value>
+                <value>
+                    <fullName>Endorsement</fullName>
+                    <default>false</default>
+                    <label>Endorsement</label>
+                </value>
+                <value>
+                    <fullName>Licensure</fullName>
+                    <default>false</default>
+                    <label>Licensure</label>
+                </value>
+                <value>
+                    <fullName>Other</fullName>
+                    <default>false</default>
+                    <label>Other</label>
+                </value>
+                <value>
+                    <fullName>Accommodations</fullName>
+                    <default>false</default>
+                    <label>Accommodations</label>
+                </value>
+                <value>
+                    <fullName>Family Considerations</fullName>
+                    <default>false</default>
+                    <label>Family Considerations</label>
+                </value>
+                <value>
+                    <fullName>International Student</fullName>
+                    <default>false</default>
+                    <label>International Student</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Contact__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
```

Comment:
@xinnanlisf do we want to set to null or do we want to block deletion?

Reply:
@lifewithryan, I think we should block deletion since it won't make sense if an attribute record is not associated with a contact.